location/services.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from django.contrib.auth.models import AnonymousUser
+from django.core.serializers.json import DjangoJSONEncoder
+
+from location.apps import LocationConfig
+
+
+def check_authentication(function):
```

Comment:
@dborowiecki I remember we discussed about this also during developing Invoice module. Your idea was to move such services 'utils' into 'core' module so as to not repeat the same code. Anyway we can keep this for that moment. But in the future we can consider moving this into core and refactor this in another modules that used this service 'utils'.

Reply:
Also we can move base services like 'add'/'update'/'delete' because in Invoice it was done by you in generic way.